tools/importer-rest-api-specs/components/discovery/interface.go

```diff
@@ -2,6 +2,8 @@ package discovery
 
 import "github.com/hashicorp/pandora/tools/sdk/config/definitions"
 
+const RootNamespace = "Pandora.Definitions.ResourceManager"
```

Comment:
this intentionally isn't a constant btw, since it's intended to only be used in this one place since this is specific to Resource Manager - when we add support for the MS Graph definitions, that'll likely become `Pandora.Definitions.MicrosoftGraph`